### DISCUSSION TITLE
Select 组件的数据更新问题

### DISCUSSION THREAD

**Discussion by hujiangang** [vbenjs/vue-vben-admin#2946] [Q&A]
一个Select组件,下拉目前如下:
 field: 'SystemVersionID',
 label: '系统版本',
 component: 'Select',
componentProps: {
      options: [
        {
          label: '测试服',
          value: '1',
          key: '1',
        }
      ],
    },

可是 服务器f返回的是 ret  -> {"PlatformVersionID":1,"SystemVersionID":1}  
直接设置 setFieldsValue(ret)
select 框中显示 1了，怎么样显示label的内容?

**Comment by DesignHhuang** [comment]
把返回的数字 1 变成字符串 '1'
